### ISSUE TITLE
Update name to reflect dependency on bootstrap 2

### ISSUE THREAD

**Issue by yauh** [JackAdams/meteor-editable-text-wysiwyg#3]
While making it more modular I suggest to reflect the dependency on bootstrap 2 in the name of this package, similar to bootstrap 3. Perhaps someone will want to adapt a package for zurb or whatever is out there.
Just a thought, albeit not a big one ;)

**Comment by JackAdams** [comment]
Agreed. I'll do that as soon as I get a spare moment. People need to know that it's pulling in bootstrap-2, which might mess up their existing styling.
